mars-cli/setup.py

```diff
@@ -30,7 +30,7 @@ def run(self):
     description="Multi-omics Adapter for Repository Submissions",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    packages=find_packages(include=["mars_lib"]),
+    packages=find_packages(include=["mars_lib", "mars_lib.models"]),
```

Comment:
Oof! How did I not see this... Good catch though!
Was thinking we could also do this:
```suggestion
    packages=find_packages(include=["mars_lib.*"]),
```
What do you think?